db/migrate_cards/data/progress_bar.json

```diff
@@ -0,0 +1,42 @@
+{
+  "url": null,
+  "timestamp": "2015-05-28 21:03:51 +0100",
+  "card": {
+    "name": "export",
+    "type": "Pointer",
+    "content": "[[progress bar]]\n[[Project]]\n[[progress bar+*right+*structure]]\n[[project task progress]]\n[[campaign+*type+*structure]]\n[[project with progress bar]]",
+    "value": [
+      {
+        "name": "progress bar",
+        "type": "Cardtype",
+        "codename": "progress_bar"
+      },
+      {
+        "name": "Project",
+        "type": "Cardtype",
+        "codename": "project"
+      },
+      {
+        "name": "progress bar+*right+*structure",
+        "type": "HTML",
+        "content": "<div class=\"progress\">\r\n  <div class=\"progress-bar\" role=\"progressbar\" aria-valuenow=\"{{_l|raw}}\" aria-valuemin=\"0\" aria-valuemax=\"100\" style=\"width: {{_l|raw}}%;\">\r\n    {{_l|raw}}%\r\n  </div>\r\n</div>"
```

Comment:
I am thinking that if the right structure can show the progress bar based on the left value then we don't need to create a type. I think it is a minimum way to show a progress bar?